l10n_it_fatturapa/models/account.py

```diff
@@ -82,7 +145,62 @@ class fatturapa_format(orm.Model):
     }
 
 
+class welfare_fund_type(orm.Model):
+    #_position = ['2.1.1.7.1']
+    _name = "welfare.fund.type"
+    _description = 'welfare fund type'
+
+    _columns = {
+        'name': fields.char('name'),
+        'description': fields.char('description'),
+    }
+
+
+class welfare_fund_data_line(orm.Model):
+    #_position = ['2.1.1.7']
+    _name = "welfare.fund.data.line"
+    _description = 'FatturaPA Welfare Fund Data'
+
+    _columns = {
+        'name': fields.many2one(
+            'welfare.fund.type', string="Welfare Fund Type"),
+        'fund_fiscalpos': fields.many2one(
```

Comment:
Vedo che in questo campo scrivi poi ```NaturaId```. Ma la gestione di ```Natura``` l'ho già aggiunta ai moduli